public/js/main.js

```diff
@@ -255,10 +255,10 @@ function setFileURLs(new_provider) {
       var description = getSafeHighlightedValue(hit._highlightResult.description);
       var row = '<tr id="' + hit.objectID + '" data-sri="' + hit.sri + '">' +
         '<td>' +
-          '<p><a itemprop="name" href="/libraries/' + hit.name + '">' +
-            hit._highlightResult.name.value +
-          '</a></p>' +
-          '<p class="text-muted">' + description + '</p>' +
+          '<a style="text-decoration: none;" itemprop="name" href="/libraries/' + hit.name + '">' +
```

Comment:
1. Can this styling be done in one of the css files, please.
2. The library name should retain the underline (I suggest wrapping in it a span to be able to do that).
3. Make the `a` an inline-block, instead of being completely inline.
4. `itemprop` should remain on the name of the library only.

Reply:
I added another commit according to the comments. Using a span was somehow removing all other elements so I just created another `a` element for the description. Could you take a look?

Reply:
So my issue with this is that this doesn't really solve the initial issue, there is still a massive dead area in the middle, which is why using a single `a` element would be the best option.

![](https://i.cdnjs.dev/7DCCkvhzV5.gif)

Could you try using a single `a` element again, if you need help with the span issue, let me know once you've committed adding the `a` back.